### PR TITLE
Update Spotify documentation including playlists

### PR DESCRIPTION
**Pull request by lindsaymarkward**
Improve English language use for readability; add example of play_media with Spotify playlist; add note about playlists not working on Google Nest Home devices.

## Proposed change

Improve documentation, including an example of playing playlists, which is useful where people are looking for it.
Also add a note about the current problem that Spotify playlists will not play directly on Google Home / Nest devices. See: https://community.home-assistant.io/t/spotify-playlist-wont-play/231822 

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/spotify.markdown [hunk 1]
```diff
@@ -72,29 +72,45 @@ client_secret:
...
+```
+
+The `media_content_id` value can be obtained from the Spotify desktop app by clicking on the more options ("...") next to the album art picture, selecting "Share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
+
+<div class='note'>
+
+  There is currently a problem targeting Google Nest Home media player devices (in `entity_id`). 
+  These do not currently play Spotify playlists, but other controls do work.
+
+</div>
```
Comment (frenck) [inline source/_integrations/spotify.markdown]: This is not related to this integration. Please remove this note.

File: source/_integrations/spotify.markdown [hunk 2]
```diff
@@ -72,29 +72,45 @@ client_secret:
...
+in the [Activating the Spotify integration](#activating-the-spotify-integration) section. 
 
-## URI Links for Playlists
+## Playing Spotify playlists
 
 You can send playlists to Spotify using the `"media_content_type": "playlist"`, which is part of the
-[media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service.
-The `media_content_id` can be obtained from the Spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
+[media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service, for example:
+```yaml
```
Comment (frenck) [inline source/_integrations/spotify.markdown:96]: - A code block needs to be surrounded by an empty new line.
- There is an empty line in the code block itself that is not needed.
- This code sample is incomplete and thus not copy-pastable.
- `alias` is not required and should be removed.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @lindsaymarkward

I've reviewed it and left some comments. Could you take a look?

Thanks 👍

File: source/_integrations/spotify.markdown (hunk 1 above)
Comment (lindsaymarkward) [reply source/_integrations/spotify.markdown]: Where could we put this instead? One of the biggest problems I have with HA documentation is that I try something; it doesn't work and then I find out on the forums that it's been a problem for a long time. This Spotify documentation is where I, as a user, would expect to find current limitations/problems with playing Spotify playlists.
Where would you suggest would be better for users looking to play playlists?

Comment (lindsaymarkward) [reply source/_integrations/spotify.markdown]: @frenck - I have removed this for now, but please consider how to better support users with "just in time" location of this sort of information.

File: source/_integrations/spotify.markdown (hunk 2 above)
Comment (lindsaymarkward) [reply source/_integrations/spotify.markdown:96]: The code block should now be a complete script with an empty line before and no `alias`. I did not see an empty line in the code block.
Hopefully this is acceptable now.

File: source/_integrations/spotify.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/spotify.markdown]: The note you are trying to add is a limitation for cast devices. Probably the `cast` integration is a better place for such a note. This note wasn't related to any of this. This integration cannot change any of that behavior either.

**Review by frenck — APPROVED** [review]
Thanks, @lindsaymarkward 👍 

Happy Hacktoberfest 🎉
